sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -4,9 +4,9 @@
 
 ### Features Added
 - Add a constructor to create an instance of `CopyAuthorization` model.
+- Added BinaryData support to document analysis client using `beginAnalyzeDocument(String modelId, BinaryData document, long length)`
 
 ### Breaking Changes
```

Comment:
Not related to this PR, but in the changelog you have this entry:
> Renamed `DocumentModelDetails` model to `DocumentModelInfo`

Should it be the other way around?